.env.example

```diff
@@ -10,6 +10,21 @@ TOKEN_KEY=TheTokenKey
 MAILER_API_KEY=theTokenOfTheMailerService
 MAILER_EMAIL=theEmailUsedByTheMailerAsSender
 API_BASE_URL=theApiBaseUrlToInsertInTheLinksSharedInEmails
-TOKEN_KEY=TheTokenKey
-
 BCRYPT_HASH_ROUNDS=theBcryptSaltingRounds
+
+
+
+facebookClientId=FACEBOOK_APP_ID
```

Comment:
@Ramadhan0 so that we maintain the same format as the other env variables, change these to:

```
FACEBOOK_CLIENT_ID=theFacebookClientId
GOOGLE_CLIENT_SECRET=theGoogleClientSecret
TWITTER_SESSION_KEY=theTwitterSessionKey
```

please, use this format for the rest.